salt/metalk8s/kubernetes/cni/calico/deployed.sls

```diff
@@ -779,11 +3833,10 @@ spec:
       namespace: kube-system
       labels:
         k8s-app: calico-kube-controllers
-      annotations:
-        scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
       nodeSelector:
         kubernetes.io/os: linux
+        node-role.kubernetes.io/master: ''
```

Comment:
Is this from the upstream manifest, or added by us?

Reply:
Added by me

Reply:
#2827